Kernel/CMakeLists.txt

```diff
@@ -82,9 +82,9 @@ set(KERNEL_SOURCES
     Graphics/Console/VGAConsole.cpp
     Graphics/DisplayConnector.cpp
     Graphics/GraphicsManagement.cpp
+    Graphics/Generic/DisplayConnector.cpp
```

Comment:
This needs to go up one line, e before r :)

Reply:
Done.